src/main/proto/netflix/titus/agent.proto

```diff
@@ -269,4 +269,54 @@ message ContainerInfo {
     }
     // Process to be executed inside the container
     optional Process process = 38;
+
+    // Container hostname (not necessarily the same as task ID)
+    optional string hostName = 39;
+
+    // Container task ID
+    optional string taskId = 40;
+
+    // Container launch timestamp, in Unix epoch seconds
+    optional uint64 launchTimeUnixSec = 41;
+}
+
+// For the identity client to query the metadata server to confirm the
+// currently running task's identity
+message TaskIdentity {
+    // Current state of the task
```

Comment:
I'd be reluctant to do required, since I think we intend to move to protobuf v3 at some point.